src/components/ImageLink.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+const ImageLink = ({ to, src }) => (
+  <Link to={to}>
+    <img src={src} alt="Link" />
```

Comment:
This alt could be more descriptive, maybe could even make it to a required prop?